signature/src/signatures/sigstore.rs

```diff
@@ -163,4 +163,137 @@ pub fn get_sigs_from_specific_sigstore(sigstore_uri: url::Url) -> Result<Vec<Vec
     }
 
     Ok(res)
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use oci_distribution::Reference;
+    use std::fs;
+    use std::env;
+    use std::path::{Path, PathBuf};
+
+    use image::digest::Digest;
+
+    #[test]
+    fn test_get_sigs_from_specific_sigstore() {
+        let current_dir = env::current_dir().expect("not found path");
+        let test_sigstore_dir = format!("file://{}/fixtures/signatures", current_dir.to_str().unwrap());
+        let test_sigstore_uri = url::Url::parse(test_sigstore_dir.as_str()).unwrap();
+        assert!(get_sigs_from_specific_sigstore(test_sigstore_uri.clone()).is_ok());
+        assert_eq!(
+            2,
+            get_sigs_from_specific_sigstore(test_sigstore_uri)
+                .unwrap()
+                .len()
+        );
+    }
+
+    #[test]
+    fn test_get_sigstore_base_url() {
+        #[derive(Debug)]
+        struct TestData<'a> {
+            reference: Reference,
+            sigstore_base_url: &'a str,
+        }
+
+        let tests = &[
+            TestData {
+                reference: Reference::try_from("example1.com/mylib/busybox:latest").unwrap(),
+                sigstore_base_url: "file:///var/lib/containers/sigstore",
+            },
+            TestData {
+                reference: Reference::try_from("example3.com/mylib/busybox:latest").unwrap(),
```

Comment:
Here `sigstore_base_url` is the default base url because there isn't match with `example3.com` namespace. It took me a while to understand it...maybe worth a comment?